tests/unit/work-items/work-items.test.ts

```
@@ -82,6 +82,21 @@ describe(!!(config.workspaceSlug && config.projectId && config.userId), "Work It
     expect(foundWorkItem).toBeDefined();
   });
 
+  it("should list work items with pql filter", async () => {
+    const allWorkItems = await client.workItems.list(workspaceSlug, projectId);
+    const priority = allWorkItems.results[0]?.priority ?? "none";
+
+    const filtered = await client.workItems.list(workspaceSlug, projectId, {
+      pql: `priority IN ("${priority}")`,
+    });
+
+    expect(filtered).toBeDefined();
```

ESLint

[error] 'expect' is not defined.

(no-undef)